src/dict.mjs

```diff
@@ -792,12 +792,12 @@ function withoutCollision(root, key) {
 /**
  * @template K,V
  * @param {undefined | Node<K,V>} root
- * @param {(value:V,key:K)=>void} fn
- * @returns {void}
+ * @param {(value:V,key:K)=>boolean|void} fn
```

Comment:
Shouldn't this be `(value:V,key:K)=>boolean` without the `|void`?

Reply:
The function isn't required to return a boolean, but if it does return false then the iteration will terminate at that point.